src/main/java/com/wbigelow/ancillary/modules/AnonymousMessagingModule.java

```diff
@@ -49,9 +47,20 @@
      * @return the created ID.
      */
     private int createAnonIDForUser(final MessageAuthor user) {
-        int userID = RANDOM.nextInt(MAX_ID);
-        while (anonIDs.inverse().containsKey(userID)) {
-            userID = (userID + 1) % MAX_ID; // Linear probing to find an unused ID.
+        int userID;
+        if (anonIDs.keySet().size() > MAX_ID) { // We've filled up on IDs presumably
```

Comment:
Doesn't this need to be `>=`? 
```suggestion
        if (anonIDs.keySet().size() >= MAX_ID) { // We've filled up on IDs presumably
```

Reply:
oh yup